custom-recipes/api-connect/recipe.json

```diff
@@ -5,6 +5,7 @@
         "icon": "icon-rocket"
     },
     "kind": "PYTHON",
+    "selectableFromDataset": "input_A_role",
     "inputRoles": [
         {
             "name": "input_A_role",
```

Comment:
could you change the "name" variable to something more meaningful?

Reply:
I'd like to, but I've run some tests and changing this corrupts existing flows...